components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -23,31 +23,37 @@
 
 #include <stdlib.h>
 #include <string.h>
+
 #include "psa_crypto_srv_ifs.h"
 #include "psa/client.h"
 #include "crypto.h"
 #include "crypto_platform_spe.h"
 
+#include "mbed_assert.h"
+
 #define MINOR_VER 1
```

Comment:
aren't we changing minor version with this commit? need to be careful here for bc

Reply:
no need